fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzeStmtAnalyzer.java

```diff
@@ -145,6 +150,29 @@ public Void visitAnalyzeStatement(AnalyzeStmt statement, ConnectContext session)
                 statement.setPartitionIds(pidList);
             }
 
+            // ANALYZE TABLE xxx
+            // ANALYZE TABLE xxx ALL COLUMNS
+            if (statement.isAllColumns() && CollectionUtils.isEmpty(columns)) {
+                List<String> collectibleColumns = StatisticUtils.getCollectibleColumns(analyzeTable);
+                statement.setColumnNames(collectibleColumns);
+            }
+
+            // ANALYZE TABLE xxx PREDICATE COLUMNS
+            if (statement.isUsePredicateColumns()) {
+                List<String> targetColumns = Lists.newArrayList();
+
+                List<ColumnUsage> predicateColumns =
+                        PredicateColumnsMgr.getInstance().queryPredicateColumns(statement.getTableName());
+                for (ColumnUsage col : ListUtils.emptyIfNull(predicateColumns)) {
+                    Column realColumn = analyzeTable.getColumnByUniqueId(col.getColumnFullId().getColumnUniqueId());
```

Comment:
is  `ColumnUniqueId`  changeable if it's rename or dropped & added?

Reply:
`ColumnUniqueId` is immutable & monotonic-incremented id for a table
- rename: unchanged
- dropped & added: a new id